### PR TITLE
Add max connection to variable

### PR DESCRIPTION
**Pull request by shibaPuppy**
should be able to change the max connection

reopen - https://github.com/metal3-io/mariadb-image/pull/5

### CODE REVIEW

**Comment by shibaPuppy** [comment]
cc. @bfournie @derekhiggins @dtantsur @elfosardo @kashifest

File: runmariadb [hunk 1]
```diff
@@ -10,6 +10,7 @@ MARIADB_CERT_FILE=/certs/mariadb/tls.crt
 MARIADB_KEY_FILE=/certs/mariadb/tls.key
 RESTART_CONTAINER_CERTIFICATE_UPDATED=${RESTART_CONTAINER_CERTIFICATE_UPDATED:-"false"}
 USER="$(whoami)"
+MARIADB_MAX_CONNECTIONS=${MARIADB_MAX_CONNECTIONS:-"1024"}
```
Comment (elfosardo) [inline runmariadb]: so we're changing default from 64 to 1024?

Comment (dtantsur) [reply runmariadb]: No, we're definitely not. I objected to it in the previous PR, and I'm very disappointed to see the same ridiculously high value reintroduced here without an explanation.

**Review by dtantsur — CHANGES_REQUESTED** [review]
Please match the MariaDB default or our old default.

File: runmariadb (hunk 1 above)
Comment (shibaPuppy) [reply runmariadb]: @dtantsur @elfosardo 
sorry
i forgot to fix it again during the rebase merge.
changed to the same value as before

File: README.md [hunk 2]
```diff
@@ -12,3 +12,4 @@ functionality:
 - `MARIADB_CERT_FILE` and `MARIADB_KEY_FILE` - Paths to the TLS certificates.
    Default to `/certs/mariadb/tls.crt` and `/certs/mariadb/tls.key`.
    TLS is enabled if these exist.
+- `MARIADB_MAX_CONNECTIONS` - maraiaDB max connection option
```
Comment (elfosardo) [inline README.md]: nit: mariaDB is spelled wrong

Comment (shibaPuppy) [reply README.md]: @elfosardo 
thx, fix typo 👍

File: runmariadb [hunk 3]
```diff
@@ -32,7 +33,7 @@ if [[ -f "$MARIADB_CERT_FILE" && "${RESTART_CONTAINER_CERTIFICATE_UPDATED}" == "
 fi
 
 if [ ! -d "${DATADIR}/mysql" ]; then
-    crudini --set "$MARIADB_CONF_FILE" mysqld max_connections 64
+    crudini --set "$MARIADB_CONF_FILE" mysqld max_connections "$MARIADB_MAX_CONNECTIONS"
```
Comment (tuminoid) [inline runmariadb]: Please add curlys braces to follow convention.

File: README.md [hunk 4]
```diff
@@ -12,3 +12,4 @@ functionality:
 - `MARIADB_CERT_FILE` and `MARIADB_KEY_FILE` - Paths to the TLS certificates.
    Default to `/certs/mariadb/tls.crt` and `/certs/mariadb/tls.key`.
    TLS is enabled if these exist.
+- `MARIADB_MAX_CONNECTIONS` - mariaDB max connection option
```
Comment (tuminoid) [inline README.md]: Proper capitalization.

```suggestion
- `MARIADB_MAX_CONNECTIONS` - MariaDB max connection option
```

**Review by tuminoid — COMMENTED** [review]
Nits

File: runmariadb (hunk 3 above)
Comment (shibaPuppy) [reply runmariadb]: thx, add curlys braces

**Comment by shibaPuppy** [comment]
@derekhiggins @elfosardo 
why did the merge fail?

**Comment by derekhiggins** [comment]
From above
> Merging can be performed automatically once the requested changes are addressed.

I guess @dtantsur has to confirm that the changes he requested were done

**Comment by tuminoid** [comment]
/hold
Until issues solved, otherwise this blocks tide merging other PRs

**Comment by shibaPuppy** [comment]
@derekhiggins @tuminoid @elfosardo 
https://github.com/metal3-io/mariadb-image/pull/12
I couldn't find the cause, so I created a new PR.

**Pull request closed without merging**